src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -7215,12 +7206,6 @@ int wm_vuldet_select_scan_type(char *agent_id, time_t ignore_time, char *hotfix_
         goto end;
     }
 
-    if ((obj_it = obj->child) &&
-        (obj_it = cJSON_GetObjectItem(obj_it, "hotfix_scan_id")) &&
```

Comment:
We decided to modify the queries together with the **agents** schema. And the modification of the schema requires a new schema upgrade.
As this is only a fix for the Windows scan, and this schema modification is performed in the [epic 7749](https://github.com/wazuh/wazuh/issues/7749) together with other several modifications of the DB, we decided to keep the schema and the queries until the integration of this epic to avoid generating a DB update only for this fix.